spec/integration/lib/auth0/api/v2/api_stats_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+describe Auth0::Api::V2::Stats do
+  let(:client) { Auth0Client.new(v2_creds) }
+
+  describe '.active_users' do
+    let!(:active_users) { client.active_users }
+    it { expect(Integer(active_users)).to be >= 0 }
+  end
+
+  describe '.daily_stats' do
+    let(:from) { Time.zone.today.prev_day.strftime('%Y%m%d') }
```

Comment:
I think is more clear to create a date from and another to, instead of use fix dates or all in one line.